lld/COFF/Config.h

```diff
@@ -130,9 +130,9 @@ struct Configuration {
   bool forceMultipleRes = false;
   bool forceUnresolved = false;
   bool debug = false;
-  bool debugDwarf = false;
+  bool includeDwarfChunks = false;
   bool debugGHashes = false;
-  bool debugSymtab = false;
+  bool writeSymtab = false;
```

Comment:
isn't `writeDebugSymtab` a better name here?

Reply:
I dunno - the symtab itself is entirely separate from debug info (even if one can't really use it for much anything else than help for inspecting things and debugging).